src/pages/dataset-registration-page/form-informationmodel/external-information-models/external-information-models.component.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import InputField from '../../../../components/fields/field-input/field-input.component';
+import MultilingualField from '../../../../components/multilingual-field/multilingual-field.component';
+import LinkReadonlyField from '../../../../components/fields/field-link-readonly/field-link-readonly.component';
+import InputFieldReadonly from '../../../../components/fields/field-input-readonly/field-input-readonly.component';
+import { datasetFormPatchThunk } from '../../formsLib/asyncValidateDatasetInvokePatch';
+import { getConfig } from '../../../../config';
+
+import '../form-informationmodel.component.scss';
+
+const ExternalInformationModels = ({
+  fields,
+  titleLabel,
+  linkLabel,
+  addLabel,
+  languages,
+  isReadOnly,
+  dispatch,
+  catalogId,
+  datasetId
+}) => {
+  const onDeleteFieldAtIndex = (fields, index) => {
+    const values = fields.getAll();
+    // use splice instead of skip, for changing the bound value
+    values.splice(index, 1);
+    const patch = { [fields.name]: values };
+    const thunk = datasetFormPatchThunk({ catalogId, datasetId, patch });
+    dispatch(thunk);
+  };
+
+  const isFdkURI = (uri) => uri && uri.includes(`${getConfig().searchHost}/informationmodels/`);
+
+  return (
+  <div>
+    {fields &&
+      fields.map((item, index) => (
+        <div key={index} className={isFdkURI(fields.get(index).uri) ? "display-none" : "d-flex mb-2"} >
```

Comment:
Index skal ikke brukes som nøkkel. React bruker det for å bestemme om noe skal rerendres og derfor forventer noe stabilt og unikt som identifiserer et element

Reply:
c6e505b